### DISCUSSION TITLE
All users show as ACTIVE(Even those who have never entered and are blocked)

### DISCUSSION THREAD

**Discussion by dima-online** [requarks/wiki#3736] [Error / Bug Report]
On my server (installation as a Docker deployment), all users are marked as active, even if they have never logged into the server and are blocked.

Perhaps this arose after updating to Wiki.js 2.5.191, or perhaps it arose after changing the JWT Configuration from the default domain to the server domain.
![2021-03-21_12-12-48](https://user-images.githubusercontent.com/35094556/111896058-17b24b00-8a41-11eb-994e-f994d5faee31.png)

**Comment by NGPixel** [comment]
The "active" flag is completely unrelated to whether the user has logged once or not. It simply indicates whether the user is allowed to login or not.

**Comment by dima-online** [comment]
Guest is blocked but it show as "active"
